scripts/deploy-tasks.sh

```diff
@@ -34,12 +35,12 @@ visit "${REPO_DIR}/release/tasks"
       if [[ -z ${TEKTON_TASKS_IMAGE} ]]; then
         kubectl apply -n ${DEPLOY_NAMESPACE} -f "${TASK_NAME}.yaml"
       else
-        TASKS_IMAGE="quay.io/kubevirt/tekton-tasks:${RELEASE_VERSION}"
-        DISK_VIRT_IMAGE="quay.io/kubevirt/tekton-tasks-disk-virt:${RELEASE_VERSION}"
-
-        sed "s!${TASKS_IMAGE}!${TEKTON_TASKS_IMAGE}!g" "${TASK_NAME}.yaml" | \
-        sed "s!${DISK_VIRT_IMAGE}!${TEKTON_TASKS_DISK_VIRT_IMAGE}!g" "${TASK_NAME}.yaml"  | \
-        kubectl apply -n ${DEPLOY_NAMESPACE} -f -
+        TASKS_IMAGE="quay.io/kubevirt/tekton-tasks"
+        DISK_VIRT_IMAGE="quay.io/kubevirt/tekton-tasks-disk-virt"
```

Comment:
Why does the `RELEASE_VERSION` have to be dropped?

Reply:
because the script should replace the URL no matter the release version

Reply:
Got it! Thanks!